bedrock/pocket/templates/pocket/privacy.html

```diff
@@ -12,131 +12,129 @@
+        <li>{{ ftl('pocket-privacy-necessary') }}</li>
+        <li>{{ ftl('pocket-privacy-analytics') }}</li>
+        <li>{{ ftl('pocket-privacy-personalized') }}</li>
       </ul>
-      <h3 class="cookies-subheading">Your choices</h3>
+      <h3 class="cookies-subheading">{{ ftl('pocket-privacy-your-choices') }}</h3>
       <ul>
-        <li>Cookie Preference tool: By default, Pocket sets necessary and analytics cookies, and disables personalized advertising cookies. You can manage these cookies from this <button id="ot-sdk-btn" class="ot-sdk-show-settings">Cookie Prefrences</button> tool. <a href="https://help.getpocket.com/article/1163-pocket-website-cookies-faq">Learn more</a>.</li>
-        <li>Your browser: Your browser preferences menu can help you manage cookies and prevent cross-site tracking.</li>
-        <li>Google’s Analytics opt-out: Install the <a href="https://tools.google.com/dlpage/gaoptout">Google Analytics Opt-out Browser Add-on</a>, which prohibits data transmission to Google Analytics.</li>
-        <li>External opt-out tools: Some third- party organizations provide ways of opting out of personalized advertising delivered by participating companies. For information about how to opt-out, you can visit these organizations’ websites: <a href="https://optout.networkadvertising.org/?c=1">Network Advertising Initiative’s opt-out tools</a>, the <a href="https://optout.aboutads.info/?c=2&lang=EN">Digital Advertising Initiative’s Browser Check tool</a>, and the <a href="https://www.youronlinechoices.com/uk/your-ad-choices">European Interactive Digital Advertising Alliance Your Ad Choices</a>.</li>
-        <li>Your Right to Opt-Out of the Sale of Personal Information: Pocket does not share or sell your personal information. If you enable personalized advertising on Pocket, we allow advertisers to personalize your ads on Pocket and this may be considered a sale of your information under California’s privacy law. You can opt-out by navigating to your Cookie Preference tool → Personalized Advertising Cookies and toggling to “disabled”.</li>
+        <li>{{ ftl('pocket-privacy-cookie-preference', cookie_button_id='ot-sdk-btn', cookie_button_class='ot-sdk-show-settings', cookie_faq_link='https://help.getpocket.com/article/1163-pocket-website-cookies-faq',) }}</li>
+        <li>{{ ftl('pocket-privacy-your-browser') }}</li>
+        <li>{{ ftl('pocket-privacy-google-analytics-opt', ga_opt_out_link='https://tools.google.com/dlpage/gaoptout') }}</li>
+        <li>{{ ftl('pocket-privacy-external-opt-out', opt_out_link_1='https://optout.networkadvertising.org/?c=1', opt_out_link_2='https://optout.aboutads.info/?c=2&lang=EN', opt_out_link_3='https://www.youronlinechoices.com/uk/') }}</li>
```

Comment:
Suggestion (non-blocking): omit the `lang=EN` from the link URL here if we're going to localise this page?